lib/libzpool/uzfs_zap.c

```diff
@@ -61,8 +73,8 @@ uzfs_update_zap_entries(void *zvol, const uzfs_zap_kv_t **array,
 
 	for (i = 0; i < count; i++) {
 		kv = array[i];
-		VERIFY0(zap_update(os, ZVOL_ZAP_OBJ, kv->key, 1, kv->size,
-		    kv->value, tx));
+		VERIFY0(zap_update(os, ZVOL_ZAP_OBJ, kv->key, kv->size, 1,
```

Comment:
I understand our kv->size is 8 currently. But, its better to be (.., 1, kv->size,..) @mynktl incase if kv->size becomes greater than 8 later on. Same goes to below also.

Reply:
I kept it this way to avoid fat zap upgrade.